scalafix/rules/src/main/scala/fix/CanBuildFrom.scala

```diff
@@ -153,6 +157,8 @@ case class CanBuildFromNothing(param: Name,
       body.collect {
         case ap @ Term.ApplyType(Term.Select(e, to @ toTpe(_)), List(cc2 @ matchCC(_))) =>
 
+          handledTo += to
```

Comment:
How is this code related to the `.sameElements` rewrite rule?

Reply:
Can you explain the list of changes performed by this commit in the commit message?